ui/components/ui/form-field/index.scss

```diff
@@ -7,6 +7,7 @@
 
   .info-tooltip {
     display: inline-block;
+    margin-top: -6px;
```

Comment:
We'll want to check all the places this is used.  Also, could this be achieved with `flexbox` / `align-items` so that we can count on content being centered when on multiple lines?

Reply:
I've achieved this with `align-items: baseline` - this line has been removed

Reply:
Screenshots are in the PR of other places this is used